### PR TITLE
Added dinamic decimals

### PR DESCRIPTION
**Pull request by coolwather**

### CODE REVIEW

File: src/components/transactions/modals/ClaimWRBTCModal.vue [hunk 1]
```diff
@@ -195,13 +196,13 @@ export default {
...
           const estimatedGas = new BigNumber(this.responseEstimatedGas?.amount)
-            .shiftedBy(-8)
+            .shiftedBy(decimals)
             .toString()
 
           this.sharedState.web3.eth.getGasPrice().then(gasPrice => {
             const costInWei = new BigNumber(estimatedGas)
               .multipliedBy(gasPrice)
-              .shiftedBy(-10)
+              .shiftedBy(decimals)
```
Comment (pmprete) [inline src/components/transactions/modals/ClaimWRBTCModal.vue:205]: The -10 was incorrect here right?

**Review by pmprete — APPROVED** [review]
If the -10 was incorrect then it's ready to merge, if not, there should something like SideTokenDecimals - originalDecimals

File: src/components/transactions/modals/ClaimWRBTCModal.vue (hunk 1 above)
Comment (aguschirico) [reply src/components/transactions/modals/ClaimWRBTCModal.vue:205]: Nope this isn’t correct. The token to take the decimals is the origin token, not the destination one

Comment (aguschirico) [reply src/components/transactions/modals/ClaimWRBTCModal.vue:205]: And the output token has 18 decimals in this case

**Review by aguschirico — CHANGES_REQUESTED** [review]
Please take into account that the origin token can have different amount of decimals, and you are using the destination token. Have you even tried this against the api?

**Pull request merged**